modules/behavior/draw_way.js

```diff
@@ -73,7 +75,9 @@ export function behaviorDrawWay(context, wayId, index, mode, startGraph) {
     function move(datum) {
         context.surface().classed('nope-disabled', d3_event.altKey);
 
+        var _activeLayer = _find( Hoot.layers.loadedLayers, function(a,b) { return a.activeLayer; });
         var targetLoc = datum && datum.properties && datum.properties.entity && datum.properties.entity.loc;
```

Comment:
I think we need to implement the commented out piece here, using the same split stuff we do in the draw.js file.